astronomer/airflow/version_check/update_checks.py

```diff
@@ -381,9 +382,19 @@ class UpdateAvailable(BaseApi):
         resource_name = "update_available"
         csrf_exempt = False
         base_permissions = ['can_dismiss']
+        # A bug in Flask-AppBuilder mandates us to have a value for method_permission_name
+        # before is_item_public filter can be used.
+        method_permission_name = {
+            "dismiss": "dismiss",
+        }
+        allow_browser_login = True
 
         @expose("<path:version>/dismiss", methods=["POST"])
-        @protect(allow_browser_login=True)
+        @auth.has_access(
+            [
+                (permissions.ACTION_CAN_READ, permissions.RESOURCE_ROLE),
```

Comment:
Can we use whatever is under `can dismiss on UpdateAvailable` instead?

Reply:
Not sure how to get it

Reply:
Will check tomorrow

Reply:
It also seems like it won’t be possible except we create the permission and add it to Admin

Reply:
Turns out that ('can_dismiss', 'update_available') works.

Reply:
This works `('can_dismiss', 'UpdateAvailable')`